demos/form-control/calendar.php

```diff
@@ -35,32 +35,32 @@
     Form\Control\Calendar::class,
     'type' => 'date',
     'caption' => 'Allow input (F d, Y)',
-    'options' => ['allowInput' => true]])
-     ->set(date(Date::getFormat('date')));
+    'options' => ['allowInput' => true], ])
```

Comment:
end `]` on new line, if there is NL after `[`

Reply:
fix